client/js/states/random.js

```diff
@@ -12,8 +12,46 @@ class RandomEvents {
 	}
 	// selects one of the three random events to happen
 	select() {
-		invokeRandom(this.thieves, this.brokenWagon, this.wrongPath);
+		//this.luckyWagon();
+
+		invokeRandom(this.thieves, this.brokenWagon, this.wrongPath, this.foodSpoil, this.lostOxen);
```

Comment:
Another solution is to update invokeRandom to make the first parameter an object (this), and do .call(this) on the functions